source/options.html

```diff
@@ -19,7 +19,7 @@
 			name="token"
 			spellcheck="false"
 			autocomplete="off"
-			pattern="[\da-f]{40}"
+			pattern="gp1_[\da-zA-Z_]{36,251}"
```

Comment:
```suggestion
			pattern="gp1_\w{36,251}"
```

Reply:
Btw, I don't think the existing tokens will be invalidated when the new ones start being emitted, so this should accept both, by prepending `\w{40}|` to the pattern.

Reply:
The existing format is `[\da-f]{40}` so should that be prepended instead of `\w{40}|` ?

Reply:
That’s right

Reply:
Oh, sorry, I misread the diff. Yeah, `[\da-f]{40}|` would be the correct syntax.